libs/accounts/recovery-phone/src/lib/recovery-phone.manager.ts

```diff
@@ -19,9 +19,17 @@ import {
   RecoveryNumberNotExistsError,
 } from './recovery-phone.errors';
 import { Redis } from 'ioredis';
+import { PhoneNumberInstance } from 'twilio/lib/rest/lookups/v2/phoneNumber';
 
 const RECORD_EXPIRATION_SECONDS = 10 * 60;
 
+/**
+ *
+ */
+export type PhoneNumberLookupData = ReturnType<
+  typeof PhoneNumberInstance.prototype.toJSON
+>;
```

Comment:
...isn't JSON always a string? 🤔

Reply:
Not in this case... that's what's funny about it. It gives you a JSON safe object.